screensharing/main.js

```diff
@@ -72,16 +72,16 @@ class ScreenShareMainHook {
         this._screenShareTracker = new electron.BrowserWindow({
             height: TRACKER_SIZE.height,
             width: TRACKER_SIZE.width,
-            x:(display.bounds.width - TRACKER_SIZE.width) / 2,
-            y:display.bounds.height - TRACKER_SIZE.height - 10,
+            x:(display.workArea.width - TRACKER_SIZE.width) / 2,
+            y:display.workArea.height - TRACKER_SIZE.height - 10,
             transparent: true,
             minimizable: true,
             maximizable: false,
             resizable: false,
             alwaysOnTop: true,
             fullscreen: false,
             fullscreenable: false,
-            skipTaskbar: true,
+            skipTaskbar: false,
```

Comment:
why is this change needed?

Reply:
On windows if you click hide on screen share tracker, it won't appear as a minimised window in the task bar, so basically you won't be able to bring it up.
Chrome does the same thing with their screen share track window.

Reply:
Makes sense, thanks for explaining!

Reply:
On mac this doesn't seem to have any effect.